packages/manager/modules/vps/src/dashboard/migrate/migrate.routing.js

```diff
@@ -31,15 +33,24 @@ export default /* @ngInject */ ($stateProvider) => {
         $state,
         atInternet,
         migrationTrackingPrefix,
-      ) => () => {
+      ) => (selectedPlan) => {
         atInternet.trackClick({
           name: `${migrationTrackingPrefix}::confirm`,
           type: 'action',
         });
-        return $state.go('vps.detail.dashboard.migrate.confirm');
+        return $state.go('vps.detail.dashboard.migrate.confirm', {
+          selectedPlan,
+        });
+      },
+      newPlans: /* @ngInject */ (catalog, vpsMigration) => {
+        const catalogP = [];
+        vpsMigration.plans.forEach((plans) =>
+          catalogP.push(
+            catalog.plans.find(({ planCode }) => plans.newPlan === planCode),
+          ),
+        );
+        return catalogP;
```

Comment:
```suggestion
       return vpsMigration.plans.map((plan) => catalog.plans.find(({ planCode }) => plans.newPlan === planCode))
```
forEach + push = map :)

Reply:
Okay will add the suggested lines. Thank you Marie